sql/sys_vars.cc

```diff
@@ -7350,6 +7349,16 @@ static bool check_set_default_table_encryption_access(
   // the value is unchanged.
   longlong previous_val = thd->variables.default_table_encryption;
   longlong val = (longlong)var->save_result.ulonglong_value;
+
+#ifdef WITH_WSREP
+  if (val > 1) {
```

Comment:
Can we use ENUM instead of numeric 1 or better readability and ensuring it is always blocked.

Reply:
There is no enum - I can create one, but it would still be a duplication of the existing string array.